packages/synapse-react-client/src/components/Forum/DiscussionReply.tsx

```diff
@@ -46,8 +48,22 @@ export function DiscussionReply(props: DiscussionReplyProps) {
 
   const isCurrentUserAuthor = reply.createdBy == currentUserProfile?.ownerId
 
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search)
+    const replyId = params.get('replyid')
+    if (replyId === reply.id && replyRef.current) {
+      replyRef.current.style.transition = 'background-color 1s ease'
+      replyRef.current.style.backgroundColor = '#fbf4e0'
+      setTimeout(() => {
+        if (replyRef.current) {
+          replyRef.current.style.backgroundColor = 'transparent'
+        }
+      }, 2000)
+    }
+  }, [reply.id])
+
   return (
-    <div className="reply-container">
+    <div className="reply-container" id={reply.id} ref={replyRef}>
```

Comment:
Consider making the ID something like `reply_1234` so it's not just some number that could collide with something else on the page (which I concede is very unlikely)
```suggestion
    <div className="reply-container" id={`reply_${reply.id}`} ref={replyRef}>
```